bibliopixel/layout/circle.py

```diff
@@ -1,7 +1,7 @@
 from . layout import Layout
 from . geometry.circle import (
     gen_circle, calc_ring_steps, calc_ring_pixel_count,
-    pixel_positions_from_rings)
+    gen_circle_positions)
```

Comment:
This makes things clearer.  Ish.  :-D

Honestly, I still have no idea what they mean, but don't explain it now, I'll figure it out and send out a review with dox later.